doc/tech/README.md

```diff
@@ -90,23 +93,70 @@ Any specification work created within CNTT **must** conform to set of principles
 Within the framework of the Common Telecom cloud infrastructure vision, there are four levels of documents needed to describe the components, realize the practical application of the systems and qualify the resulting cloud infrastructure. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Conformance**.
 
 <p align="center"><img src="./figures/tech_scope.png" alt="scope" title="Document Types" width="100%"/></p>
-<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+<p align="center"><b>Figure 1:</b> Documentation Scope of CNTT</p>
 
 <a name="3.1"></a>
-## 3.1 Specification Types
+## 3.1 Functional Scope
+In terms of the functional scope of the CNTT documentation, in order to target the project goals as described above, we are focussed on:
+- Functional interfaces between infrastructure and workloads
+- Functional interfaces between workload management and infrastructure management
+- Functional capabilities of the cloud infrastructure
+
+Due to the close alignment with [ETSI GS NFV 002](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/002/01.02.01_60/gs_NFV002v010201p.pdf), those ETSI interfaces that are considered relevant (with notes where required) are included in the figure below.
+
+<p align="center"><img src="./figures/cntt_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> Functional Scope of CNTT</p>
+
+<a name="3.2"></a>
+## 3.2 Out of Scope Components
+While the nature of the CNTT might seem quite broad, the following areas are not at this time part of the scope of this effort.  
+- Hardware specifications: beyond the abstracted high-level CPU, memory, network interface and storage elements.  The intention is to write the documents so they are general enough that any vendor hardware can be used in a conformant implementation without making significant changes to the model.
+- Workload specifications: Other than the API interfaces when they directly need to touch the workloads themselves, the intention is to assume the workload application is a blackbox that the cloud infrastructure is providing resources to.  The majority of interactions for lifecycle management of the workloads will be through the cloud infrastructure whenever possible.
+- Company specific requirements: The CNTT documents are designed to be general enough that most operators and others in the Open Source communities will be able
```

Comment:
Text is truncated....

Reply:
thanks - fixed in https://github.com/cntt-n/CNTT/pull/1330/commits/672990a66fa6c968aa0d29f4a6977f0ff09202cb